alertproduction.tex

```diff
@@ -51,15 +51,23 @@ \subsection{Alert Generation during Commissioning}
 
 Due to the need to verify the instrument characteristics, template quality, and image differencing and Real/Bogus performance, real-time alerts will not be immediately available during the commissioning period.
 However, the incremental template generation and alert production processes will be tested and optimized during commissioning with LSSTCam, resulting in a set of prompt data products, including alerts.
-Broker teams will be given access to these alerts for development purposes as soon as they are produced to understand their characteristics and to help to validate their quality, rather than to enable rapid followup and Early Science. 
+Broker teams will be given access to these alerts for development purposes as soon as they are produced to understand their characteristics and to help to validate their quality, rather than to enable rapid followup and Early Science.
 The Alert and Prompt Product databases will be made available for query prior to the DP2 data release.
 
 During commissioning templates will be generated incrementally over the maximal sky area supported by the available observations.
 By the end of the commissioning period, coadd templates for use in difference imaging will only be available for $\approx$ 10\% of the sky.
 Generating templates over a wide area is not an explicit goal of commissioning;  however, where possible, if commissioning observations are agnostic to pointing and filter, we would endeavour to choose a pointing and filter that maximizes building templates to enable early science.
 
-Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following System First light. 
+Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following System First light.
 Once begun, Alert Production will then proceed continuously into the full LSST survey.
 Alerts generated during commissioning may be produced with higher latency.
 The commissioning period also provides an excellent opportunity to investigate how many visits in a given band are sufficient to construct a usable template.
 
+The prompt products and Alerts generated during commissioning and the first year of the LSST survey are summarized in Table~\ref{tab:prompt-products}.
```

Comment:
Alerts are prompt products, either remove 'Alerts' or rephrase to "The prompt products, including Alerts, generated during ......"

Reply:
Good, thanks! Nice to be clear on this given that we have both a prompt products database that doesn't contain Alerts, and a separate Alert DB... I'll adopt your wording, make a fresh table including Eric's edits, commit, push and merge.